### PR TITLE
Update items sorting in weekly reports email

### PR DESCRIPTION
**Pull request by marekrozmus**

### CODE REVIEW

File: server/common/utils/helpers.js [hunk 1]
```diff
@@ -492,6 +492,23 @@ const formatItems = items => list =>
     };
   });
 
+const timestampFromDateString = date =>
+  Math.round(new Date(date).getTime() / 1000);
+
+const reportDataSortFunction = (a, b) => {
```
Comment (alemikolo) [inline server/common/utils/helpers.js]: Is the word 'Function' necessary?

Comment (Ciunkos) [reply server/common/utils/helpers.js]: "Comparator" postfix would be better for this one.

File: server/common/utils/helpers.js [hunk 2]
```diff
@@ -492,6 +492,23 @@ const formatItems = items => list =>
...
+  const aTimestamp = timestampFromDateString(a.requestedAt);
+  const bTimestamp = timestampFromDateString(b.requestedAt);
+  if (aTimestamp === bTimestamp) {
+    return 0;
+  }
+
+  if (aTimestamp < bTimestamp) {
```
Comment (Ciunkos) [inline server/common/utils/helpers.js]: Consider using ternary here, `return aTimestamp < bTimestamp ? 1 : -1`.

File: server/common/utils/helpers.js [hunk 3]
```diff
@@ -492,6 +492,19 @@ const formatItems = items => list =>
     };
   });
 
+const timestampFromDateString = date =>
+  Math.round(new Date(date).getTime() / 1000);
+
+const reportDataSortComparator = (a, b) => {
```
Comment (Ciunkos) [inline server/common/utils/helpers.js]: Consider `reportDateComparator`.

**Pull request merged**